agent/lib_mongodb.c

```diff
@@ -173,7 +175,205 @@ NR_PHP_WRAPPER(nr_mongodb_operation) {
 }
 NR_PHP_WRAPPER_END
 
-void nr_mongodb_enable(TSRMLS_D) {
+#else
+
+NR_PHP_WRAPPER(nr_mongodb_operation_before) {
+    (void)wraprec;
+    nr_segment_start(NRPRG(txn), NULL, NULL);
+}
```

Comment:
i think this needs to have an autosegment like here: https://github.com/newrelic/newrelic-php-agent/blob/main/agent/lib_predis.c#L762

Reply:
It isn't clear to me from any of our code or docs what the purpose of `auto_segment` is- I'd like to understand this better before I implement around it. Do you have any resources on what `auto_segment` is intended for?

Reply:
see 26cdcf283b07ec9283a68c3c7ee469f027a66c09

Reply:
Looks good.